frontend/src/toolbar/elements/elementsLogic.ts

```diff
@@ -32,7 +33,10 @@ export const elementsLogic = kea<elementsLogicType>({
         enableInspect: true,
         disableInspect: true,
 
-        selectElement: (element: HTMLElement | null) => ({ element }),
+        selectElement: (element: HTMLElement | null, elementType?: 'heatmap' | 'inspect' | 'action') => ({
```

Comment:
Nit, it's not really element "type" here, but more like the click's "source". The element's type is still a HTML Link/Button/etc Element.